### DISCUSSION TITLE
Make docker kill timeout configurable for services which might take a longer time to kill.

### DISCUSSION THREAD

**Discussion by mt-bt** [coollabsio/coolify#3195] [Feature Requests]
Some services might take a longer time to kill. It would be great if we could configure the docker kill time argument currently set here:

https://github.com/coollabsio/coolify/blob/69fc4c7f525100c9291b75f4b9f31f92b285bf14/app/Actions/Application/StopApplication.php#L38

**Comment by peaklabs-dev** [comment]
@matthieu-beteille I will add this in https://github.com/coollabsio/coolify/pull/3024 if it makes sense. But I am implementing a multi-stop method that shuts down all containers at once (faster after one another), so a timeout of 300 (I increased it) should be enough for everything.

**Reply by mt-bt** [reply]
Sounds great, appreciate it!

**Comment by peaklabs-dev** [comment]
The next version has a fix for a large amount of containers (multi shutdown) so this is not needed anymore.
